core/src/test/scala/doric/syntax/TimestampColumnsSpec.scala

```diff
@@ -65,11 +62,13 @@ class TimestampColumnsSpec
       )
     }
 
-    it("should fail if invalid timeZone") {
-      intercept[java.time.DateTimeException](
-        df.select(colTimestamp("timestampCol").fromUtc("wrong timeZone".lit))
-          .collect()
-      )
+    if (spark.version.take(1) == "3") {
```

Comment:
```suggestion
    if (spark.version.take(1) > "2") {
```

Reply:
If we are going to compare by value we should change it to int

Reply:
```suggestion
    if (spark.version.take(1).toInt > 3) {
```